src/angularjs/src/app/places/list/places-list.controller.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * @ngdoc controller
+ * @name pfb.place.list.controller:PlaceListController
+ *
+ * @description
+ * Controller for listing analysis jobs
+ *
+ */
+(function() {
+    'use strict';
+
+    /** @ngInject */
+    function PlaceListController($state, $stateParams, $scope, Pagination, AuthService,
+                                 Neighborhood) {
+        var ctl = this;
+
+        var defaultParams = {
+            limit: null,
+            offset: null
+        };
+        var nextParams = {};
+        var prevParams = {};
+
+        initialize();
+
+        function initialize() {
+            ctl.isAdminUser = AuthService.isAdminUser();
+
+            ctl.hasNext = false;
+            ctl.getNext = getNext;
+
+            ctl.hasPrev = false;
+            ctl.getPrev = getPrev;
+            ctl.places = [];
+
+            ctl.filters = {};
+
+            getPlaces();
+        }
+
+        function getPlaces(params) {
+            params = params || $stateParams;
+            Neighborhood.query(params).$promise.then(function(data) {
```

Comment:
Was thinking we'd otherwise need to separately query job status for each neighborhood on the front end, which isn't great, or add some query parameter to optionally get latest status from the neighborhood endpoint.

Querying for jobs instead does have some implications: only neighborhoods for which at least one job has been attempted will show in the list, and all our searching/filtering by neighborhood attributes need to work on the jobs endpoint.